### ISSUE TITLE
Images have excess padding under them

### ISSUE THREAD

**Issue by turt2live** [vector-im/element-web#18307]
develop/nightly

**Comment by SimonBrandner** [comment]
![Screenshot_20210731_081154](https://user-images.githubusercontent.com/25768714/127730647-038e9799-a129-4058-9f0b-0243dd4f3ef1.png)


I don't think I can repro

**Comment by turt2live** [comment]
![image](https://user-images.githubusercontent.com/1190097/127747272-b2b97124-cf85-4043-b661-700a05060c82.png)

It appears to be wide-ish images that cause it. The blurhash takes the appropriate area, but the image does not.

**Comment by SimonBrandner** [comment]
![Screenshot_20210801_080944](https://user-images.githubusercontent.com/25768714/127761192-d6256a12-7c24-486a-95de-08e4058755ca.png)

Still nothing with 1152x648

**Comment by turt2live** [comment]
@SimonBrandner what version of Nightly are you using? I'm still seeing this on `0.0.1-nightly.2021080101` with a 1712x480 image (for example).

**Comment by SimonBrandner** [comment]
![Screenshot_20210802_085311](https://user-images.githubusercontent.com/25768714/127816766-2f2946d1-e417-4ad0-a24e-0510f5d3b7f8.png)

```
Size: 1712x480
Version: 2021080101
```

Someone else should probably try because I am probably doing something wrong

**Comment by turt2live** [comment]
figured it out: it's the thumbnail dimensions.

Source image is 1712x480 but the thumbnail is listed at 800x224 (due to the `?width=800&height=600&method=scale` params on the thumbnail request). This causes the gap because the image height and the thumbnail height are now incorrect, largely due to us not scaling images in the timeline like we used to.

Previously, we'd over-extend the image (or use the thumbnail dimensions) which would cause a slightly blurry thumbnail or an accurate tile size. My guess is with the blurhash and message bubbles work overlapping we've ended up with a regression in thumbnail handling, particularly seeing as how various CSS attributes for images have changed in recent weeks and the blurhash takes the "correct" amount of space.
